lib/tasks/hyacinth/setup.rake

```diff
@@ -32,102 +32,6 @@ namespace :hyacinth do
         end
       end
     end
-
-    desc "Set up test projects"
-    task test_projects: :environment do
```

Comment:
I removed this because I found that the test projects were just getting in the way most of the time, since all of the sample content was in the sample content project.  I can easily add a second sample project if useful though.  Or we can add it later on.

Reply:
Sounds good to me.

Reply:
The true test projects should be handled via factories anyway.